ci/scripts/notify.py

```diff
@@ -21,6 +23,38 @@
     "connector-node-integration-test": ["siyuan"],
 }
 
+INTEGRATION_TEST_MAP = {
```

Comment:
looks like can be merged with `CASES_MAP` so that we don't write the name of test cases twice,

but not a big deal if the syntax forbids to do so

Reply:
This is actually the step key, not the case name. The case name can be repeated, but the step key cannot be repeated. There is still a difference.